litestar/routes/asgi.py

```diff
@@ -33,7 +32,7 @@ def __init__(
         super().__init__(
             path=path,
             scope_type=ScopeType.ASGI,
-            handler_names=[unwrap_partial(route_handler.handler_name)],
```

Comment:
Not sure why this was here in the first place; `router_handler.handler_name` is a string